src/assessments/contrast/test-steps/state-changes.tsx

```diff
@@ -17,28 +17,37 @@ import { ReportInstanceField } from '../../types/report-instance-field';
 import { Requirement } from '../../types/requirement';
 import { ContrastTestStep } from './test-steps';
 
-const description: JSX.Element = (
-    <span>If state changes are indicated solely by changes in color, the different states must have sufficient contrast.</span>
-);
+const description: JSX.Element = <span>Any visual effect that indicates a component's state must have sufficient contrast.</span>;
 
 const howToTest: JSX.Element = (
     <div>
         <p>The visual helper for this requirement highlights links, native widgets, and custom widgets in the target page.</p>
         <ol>
             <li>
-                In the target page, examine each highlighted element in each of the following states:
+                In the target page, examine each highlighted element to determine whether it can adopt any of the following states:
                 <ol>
-                    <li>Normal</li>
                     <li>Focused</li>
                     <li>Mouseover</li>
-                    <li>Selected (if applicable)</li>
+                    <li>Selected</li>
+                </ol>
+            </li>
+            <li>
+                Identify any visual (non-text) indicators that communicate:
+                <ol>
+                    <li>The boundary of the component's clickable area (such as a border or background color)</li>
+                    <li>The component's current state (such as a different background color or a checkmark)</li>
                 </ol>
             </li>
             <li>
-                If any component indicates state changes solely by a change of color, use <WindowsContrastCheckerAppLink /> to verify that
-                the contrast ratio between different states is at least 3:1. (If you are testing on a Mac, you can use the{' '}
-                <MacContrastCheckerAppLink />
-                .)
+                Use <WindowsContrastCheckerAppLink /> (or if you are testing on a Mac, the <MacContrastCheckerAppLink />) to verify that the
+                visual indicator has a contrast ratio of at least 3:1 against the adjacent background.
+            </li>
```

Comment:
This code matches the suggested new text from #1204, but in most of the similar "How to test" text from other requirements, I think an "Exception" block like this is usually written as a second paragraph within the preceding `<li>`, not as a separate `<li>`. @ferBonnin / @LiLoDavis, should we collapse the `Exception:` paragraph into the same `<li>` as the `Use Accessibility Insights for Windows` one for consistency?

Reply:
@dbjorge agree. This should be within the <li> and not as a separate list item.
Thanks for the contribution Isadu!